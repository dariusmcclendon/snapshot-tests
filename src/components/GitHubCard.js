import Card from 'react-bootstrap/Card'

function GitHubCard(){
    return (
        <Card>
            <Card.Img/>
            <Card.Body>
                <Card.Title>Darius McClendon</Card.Title>
                <Card.Text>
                    The first lesson as a technician is to never trust a previous technician's work.
                </Card.Text>
            </Card.Body>
        </Card>
    )
}

export default GitHubCard