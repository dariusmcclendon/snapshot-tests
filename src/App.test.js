import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer'
import App from './App';
import GitHubCard from './components/GitHubCard'
test('renders a snapchot', ()=>{
  let tree = renderer.create(<App/>).toJSON()
  expect(tree).toMatchSnapshot()
})
test('snapshot of GitHubCard', ()=>{
  let tree = renderer.create(<GitHubCard/>).toJSON()
  expect(tree).toMatchSnapshot()
})