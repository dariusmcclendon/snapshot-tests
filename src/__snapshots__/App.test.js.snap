// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapchot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;

exports[`snapshot of GitHubCard 1`] = `
<div
  className="card"
>
  <img
    className="card-img"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Darius McClendon
    </div>
    <p
      className="card-text"
    >
      The first lesson as a technician is to never trust a previous technician's work.
    </p>
  </div>
</div>
`;
